data_prep/introspector.py

```diff
@@ -452,32 +453,46 @@ def get_project_funcs(project_name: str) -> Dict[str, List[Dict]]:
   return fuzz_target_funcs
 
 
+def _parse_arguments() -> argparse.Namespace:
+  """Parses command line args."""
+  parser = argparse.ArgumentParser(
+      description='Parse arguments to generate benchmarks')
+
+  parser.add_argument('project', help='Name of the project', type=str)
+  parser.add_argument('-m',
+                      '--max-functions',
+                      type=int,
+                      default=3,
+                      help='Number of benchmarks to generate')
+  parser.add_argument('-o',
+                      '--out',
+                      type=str,
+                      default='',
+                      help='Output directory')
+
+  args = parser.parse_args()
+  return args
+
+
 if __name__ == '__main__':
   logging.basicConfig(level=logging.INFO)
 
-  # TODO(Dongge): Use argparser.
-  cur_project = sys.argv[1]
-  max_num_function = 3
-  if len(sys.argv) > 2:
-    max_num_function = int(sys.argv[2])
-  if len(sys.argv) > 3:
-    outdir = sys.argv[3]
-    os.makedirs(outdir, exist_ok=True)
-  else:
-    outdir = ''
+  args = _parse_arguments()
+  if args.out != '':
```

Comment:
Nitpicking: How about `if args.out:` as we have specified its type to `str` above?

Reply:
Sounds good, done!